blog/content/post/2023-12-14-aem_phase3.md

```diff
@@ -0,0 +1,318 @@
+    Event: 0101800003000b620444080020ef9a26fc22d1ae8cecff59e9481ac1ec533dbe228bec6d17930f4cb2cc5b972400680101800001000b62042c040020b75ce1946f78df8baa426918db09318017e6b38d048c954e05c2c4f34bd44060004600
+Entry 11:
+    PCR:        18
+    Event Type: 0x410
+    Digests:
+      SHA1: e75b9b930232d8061f2a637e6a59501a418bd3da
+      SHA256: 3b5f9781c4347f5d3af1daf7019654dc05d1af857a598c18f3bc77e7b042b37f
+    Event: 
+Entry 12:
+    PCR:        18
+    Event Type: 0x40b
+    Digests:
+      SHA1: 9069ca78e7450a285173431b3e52c5c25299e473
+      SHA256: df3f619804a92fdb4057192dc43dd748ea778adc52bc498ce80524c014b81119
+    Event: 00000000
+Entry 13:
+    PCR:        18
+    Event Type: 0x40f
+    Digests:
+      SHA1: 246445653fc4dcf9efd5c7e88036f34bebda30f3
+      SHA256: 5c09817a86d6dce28033e2fa3b6c2d9b292548f8ad06121032a187647eaf1150
+    Event: 26020000
+Entry 14:
+    PCR:        18
+    Event Type: 0x40c
+    Digests:
+      SHA1: 0aaf76f425c6e0f43a36197de768e67d9e035abb
+      SHA256: 26b25d457597a7b0463f9620f666dd10aa2c4373a505967c7c8d70922a2d6ece
+    Event: 02000000
+Entry 15:
+    PCR:        18
+    Event Type: 0x413
+    Digests:
+      SHA1: 5ba93c9db0cff93f52b521d7420e43f6eda2784f
+      SHA256: 6e340b9cffb37a989ca544e6bb780a2c78901d3fb33738768511a30617afa01d
+    Event: 
+Entry 16:
+    PCR:        18
+    Event Type: 0x414
+    Digests:
+      SHA1: 5dfea6ad59169b181f5cbc08e84455fa0e289161
+      SHA256: 8e22a74383846c88d1b09099ae44bfb1b2d983be6daef20883fd9ffb9f4babd6
+    Event: 0101800003000b620444080020ef9a26fc22d1ae8cecff59e9481ac1ec533dbe228bec6d17930f4cb2cc5b972400680101800001000b62042c040020b75ce1946f78df8baa426918db09318017e6b38d048c954e05c2c4f34bd44060004600
+Entry 17:
+    PCR:        18
+    Event Type: 0x502
+    Digests:
+      SHA1: bac7df776a03c276f35e603ce0cfc4fee9068eb7
+      SHA256: 9d1e2290b628e3d676b9d6141373f84c63d48264bea634518d3fa25134108c18
+    Event: 
+Entry 18:
+    PCR:        18
+    Event Type: 0x502
+    Digests:
+      SHA1: 3b5432648ef18ec462425018faf869b0f8f000f5
+      SHA256: 32e56f2deb41d7a5eb5b0ae176d28c79704e781aade583faf19df1ad090e1e3d
+    Event: 
+Entry 19:
+    PCR:        18
+    Event Type: 0x502
+    Digests:
+      SHA1: 2a967d058b9b05947a45eecc7620126af10b95da
+      SHA256: c65dbff6e597353bd110c84a032baec0e986c997f110c9a21a11866ea9eda393
+    Event: Measured TXT OS-MLE data
+Entry 20:
+    PCR:        17
+    Event Type: 0x502
+    Digests:
+      SHA1: 36bb39b7fe9f45cf3dc1c8eff1f471601d18c2b4
+      SHA256: 93d3fda97f1bc58402526248e59edbecf32d058f9a372dc43c9e55a777d60812
+    Event: Measured MB2 module
+Entry 21:
+    PCR:        17
+    Event Type: 0x502
+    Digests:
+      SHA1: 501fe83e26ed23d319dd61dd8dfe347e32dc1ae5
+      SHA256: f4d74eee90a45cbc5d75a98dd5d688b7fd3635bb44e7afd46990df67561d6da7
+    Event: Measured MB2 module
+```
+
+Digests from the entries can be extended on debug PCR 16 (make sure it is reset
```

Comment:
This paragraph might sound better if it starts with "If you want to validate PCR values" or "To verify PCR values are correct" because it took me a moment to understand why PCR 16 was mentioned here.

Reply:
https://github.com/3mdeb/news-and-ideas/pull/561/commits/d87984353d98b14ec17191cbad5b122f2ec048fa